exercises/patterns/printHollowRightTriangle/printHollowRightTriangle.js

```diff
@@ -34,14 +34,20 @@ let helpers = require('../printHelpers');
  * @param {number} height - The height of the triangle to print
  */
 function printHollowRightTriangle(height) {
-  for (let i = 0; i < height; i++) {
-    /*
-      This is your job. :)
-
-      Play with iterating from 1, 2, ..., height instead of 0, 1, 2, ..., height-1.
-      Depending on your approach, it might simplify the logic.
-    */
-
+  for (let i = 1; i < (height+1); i++) {
```

Comment:
Read the description of the exercises more closely. We gave you a helper function that prints out a single character multiple times. In the first few pattern exercises, we even made it fill-in-the-blank to drive the point home.

The goal isn't to be first to pencil's down. Take time to orient yourself.